synapse/lib/stormlib/cortex.py

```diff
@@ -1055,7 +1055,7 @@ class CortexHttpApi(s_stormtypes.Lib):
                       {'name': 'readonly', 'type': 'boolean',
                        'desc': 'Run the Extended HTTP Storm methods in readonly mode.', 'default': False},
                   ),
-                  'returns': {'type': 'http:api', 'desc': 'A new http:api object.'}}},
+                  'returns': {'type': 'http:api', 'desc': 'A new HTTP API object.'}}},
```

Comment:
The `http:api` object referenced here is  Synapse type :)

Reply:
Yea, this just ends up being the weird URL that makes an errant link that goes to `http://api`. I tweaked it to be more in line with how the cron libs look.